Source/Core/AbsyExpr.cs

```diff
@@ -3837,6 +3843,197 @@ public T Dispatch<T>(IAppliableVisitor<T> visitor)
     }
   }
 
+  public class FieldAccess : IAppliable
+  {
+    private IToken _tok;
+
+    private string _fieldName;
+    
+    private DatatypeSelector _selector;
+
+    public IToken tok
+    {
+      get
+      {
+        Contract.Ensures(Contract.Result<IToken>() != null);
+        return this._tok;
+      }
+      set
+      {
+        Contract.Requires(value != null);
+        this._tok = value;
+      }
+    }
+    
+    public string fieldName
```

Comment:
It is typical in C# to use an initial capital letter for properties. I don't remember what the rest of the Boogie source code does, but I propose change `tok`, `fieldName`, and `selector` to `Tok`, `FieldName`, and `Selector`.

Reply:
I have made the changes except for `Tok`. I see that lower case `tok` is used for the property consistently across all other equivalent classes.

Reply:
You're right. `.tok` has been an exception this rule, and it seems okay to leave it. (Any change to this in the future would be appropriate for a separate PR.)